modules/ROOT/pages/api-gateway-encryption-mule3.adoc

```diff
@@ -0,0 +1,114 @@
+= Gateway Startup Encryption in Mule 3
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Gateway startup encryption allows you to encrypt sensitive information stored by the runtime such as policies, contracts, and the required credentials to configure the connection between your Mule application and Anypoint Platform.
+
+== Configuring the Runtime to Use Encryption
+
+You can encrypt your Anypoint Platform credentials (client ID, client secret, and proxy password) using a 16-characters long encryption key, and pass this encryption key to your runtime during start-up through the `anypoint.platform.encryption_key` property so that the runtime can decrypt and use your encrypted credentials:
+
+[source]
+----
+$MULE_HOME/bin/mule start -Danypoint.platform.encryption_key=MyEncryptionKey1
+----
+
+While the runtime is starting it checks for an encryption key. Then it gets the client ID and client secret values from its `wrapper.conf` file.
+
+== Encrypting Credentials using the Agent
+
+To configure the runtime through the agent to use encryption the argument `encryption-key` with the desired key as value. The agent then encrypts the client ID and Secret.
+
+When starting the runtime, you need to pass the encryption key using the `anypoint.platform.encryption_key` property as instructed above.
+
+== Gateway Encryption Tool
+
+The gateway encryption tool provides a way to inspect the policy files that Mule runtime 3.9.3 and later encrypts, provides a way to encrypt offline policies, and pass encrypted client credentials to the runtime.
+
+The tool has two modes, `policy` and `property`.
+
+=== Policy
+
+This mode encrypts and decrypts the policy XML files.
+
+To encrypt a policy file:
+
+[source]
+----
+java -jar gateway-encryption-tool.jar policy <encrypt|decrypt> <key> <input_policy_file> <output_policy_file>
+----
+
+This tool takes the policy passed in `<input_policy_file>` and encrypts it using the encryption key passed in `<key>` and stores the result in the file in `<output_policy_file>`.
+
+[NOTE]
+This tool does not verify the validity of the policy.
+
+=== Property
+
+This mode encrypts and decrypts the values of the properties set in the `wrapper.conf` file.
+
+To encrypt a property of you configuration file:
+
+[source]
+----
+java -jar gateway-encryption-tool.jar property <encrypt|decrypt> <key> <input_value>
+----
+
+This tool takes the key passed in `<key>` and use it to encrypt the value passed in `<input_value>`.
+
+When using the `decrypt` option, it is possible that your terminal might interpret some of the characters of the encrypted value. To avoid this scenario, wrap the encrypted value in simple comas:
+
+[source]
+----
+java -jar gateway-encryption-tool.jar property decrypt 1234567812345678 ‘![FefVUnvDJpbXnUvrGAEezg==]’
```

Comment:
is `’` the proper character here? It is referred as `simple coma`, but the name of such character seems to be `inverted single comma` which doesn't seem the same character as the `single quotation mark`: `'`

Can you please confirm that the character used here is correct, and also confirm its proper name? Having the unicode of that character might help identify it.